### PR TITLE
(BEDS-425) implement endpoint for mobile app bundles

### PR DESCRIPTION
**Pull request by LuccaBitfly**

### CODE REVIEW

File: backend/pkg/api/enums/enums.go [hunk 1]
```diff
@@ -268,3 +268,38 @@ func (c ChartAggregation) Duration(secondsPerEpoch uint64) time.Duration {
...
+	}
+}
+
+var Environments = struct {
+	Production Environment
+	Staging    Environment
+}{
+	EnvironmentProduction,
+	EnvironmentStaging,
+}
```
Comment (invis-bitfly) [inline backend/pkg/api/enums/enums.go:305]: maybe combine with `deployment_type` from config so we dont have multiple variants/spellings through the codebase?

Comment (LuccaBitfly) [reply backend/pkg/api/enums/enums.go]: removed environment param

File: backend/pkg/api/handlers/internal.go [hunk 2]
```diff
@@ -461,6 +461,50 @@ func (h *HandlerService) InternalGetValidatorDashboardRocketPoolMinipools(w http
...
+		handleErr(w, r, err)
+		return
+	}
+	var data types.MobileBundleData
+	data.HasNativeUpdateAvailable = stats.MaxNativeVersion > nativeVersion
+	// if given bundle version is smaller than the latest and delivery count is less than target count, return the latest bundle
+	if force || (bundleVersion < stats.LatestBundleVersion && (stats.TargetCount == 0 || stats.DeliveryCount < stats.TargetCount)) {
+		data.BundleUrl = stats.BundleUrl
+	}
+	returnOk(w, r, data)
```
[the remark this answers is not in the document]
Comment (LuccaBitfly) [reply backend/pkg/api/handlers/internal.go]: fixed

**Review by remoterami — APPROVED** [review]
lgtm

**Pull request merged**